include/mbedtls/pk.h

```diff
@@ -968,9 +952,6 @@ int mbedtls_pk_parse_key(mbedtls_pk_context *ctx,
  *                  with mbedtls_pk_init() or reset with mbedtls_pk_free(). If you need a
  *                  specific key type, check the result with mbedtls_pk_can_do().
  *
- * \note            For compressed points, see #MBEDTLS_ECP_PF_COMPRESSED for
```

Comment:
Note: I was going to say in this case we should copy the relevant information here: parsing compressed points on curve SECP224R1 and SECP224K1 is not supported, but OTOH we're planning to remove support for those curves in 1.0, and one of them is already not supported in PSA, so we probably don't need to bother.